lms/urls.py

```diff
@@ -23,12 +23,12 @@
 # Use urlpatterns formatted as within the Django docs with first parameter "stuck" to the open parenthesis
 urlpatterns = (
     '',
-
     url(r'^$', 'branding.views.index', name="root"),   # Main marketing page, or redirect to courseware
     url(r'^dashboard$', 'student.views.dashboard', name="dashboard"),
     url(r'^update_subscription$', 'student.views.update_subscription', name="update_subscription"),
     url(r'^login_ajax$', 'student.views.login_user', name="login"),
     url(r'^login_ajax/(?P<error>[^/]*)$', 'student.views.login_user'),
+    url(r'^edevate/login/(?P<access_token>[\w]+)/$', 'student.views.edevate_login', name='edevate_login'),
```

Comment:
might be make `access_token` a POST parameter? as far as I know request path is visible to everyone even over HTTPS, and access token is supposed to be kept secret